andela/vidar-ah-backend
# #164139690 Add feature to follow and unfollow users

#### What does this PR do? 
This PR adds a feature to follow and unfollow users
#### Description of task to be completed?
* Add table `follows`
* Add endpoint to follow users
* Add endpoint to unfollow users
#### How should this be manually tested?
* Run `npm run db-migrate` to run migrations
* Run `npm start` to start the application
* Create a two users by sending two post requests to /api/v1/user
* Verify the users by clicking the link in your email
* Send a `GET` request along with the user id to` /api/v1/followuser/<id>` 
* To unfollow the user send a `GET` request along with the user id to` /api/v1/unfollowuser/<id>`
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/164139690
